source/baseObject.py

```diff
@@ -143,16 +159,24 @@ class ScriptableObject(AutoPropertyObject):
 	@type scriptCategory: basestring
 	"""
 
+	__metaclass__ = ScriptableType
+
 	def __init__(self):
 		#: Maps input gestures to script functions.
 		#: @type: dict
 		self._gestureMap = {}
 		# Bind gestures specified on the class.
+		# This includes scripts specified on decorated scripts.
+		# This does not include the gestures that are added when creating a DynamicNVDAObjectType.
 		for cls in reversed(self.__class__.__mro__):
 			try:
 				self.bindGestures(getattr(cls, "_%s__gestures" % cls.__name__))
 			except AttributeError:
 				pass
+			try:
+				self.bindGestures(cls._scriptDecoratorGestures)

```

Comment:
Consider using a class specific attribute here, like we do for `__gestures` above it. This isn't strictly necessary because you're looking at each specific class in the MRO, but it could make debugging weird if someone poked at `_scriptDecoratorGestures` for their class and wondered why it wasn't showing inherited stuff.